pulp_container/app/migrations/0020_update_push_repo_perms.py

```diff
@@ -0,0 +1,69 @@
+# Generated by Django 2.2.23 on 2021-06-01 14:53
+
+from django.contrib.auth.models import Group
+from django.db import migrations
+from guardian.shortcuts import assign_perm, get_objects_for_group
+
+
+def update_repo_perms(apps, schema_editor):
+    """ Assign push repo perms to the namespace and distribution groups. """
+    # Group = apps.get_model("auth", "Group")
+    AccessPolicy = apps.get_model('core', 'AccessPolicy')
+    push_repo_vewset = 'repositories/container/container-push'
+    dist_viewset = 'distributions/container/container'
+    namespace_vewset = 'pulp_container/namespaces'
```

Comment:
```suggestion
    namespace_viewset = 'pulp_container/namespaces'
```
and where it is used later...